packages/block-library/src/post-terms/edit.js

```diff
@@ -94,6 +94,16 @@ export default function PostTermsEdit( {
 					} }
 					help={ __( 'Enter character(s) used to separate terms.' ) }
 				/>
+				<ToggleControl
```

Comment:
I am not sure that this needs to be in the Advanced panel, but let's hear what others think.
For example, the post title and site title link options are in the regular block settings panel.

Reply:
@carolinan, since the separator is still something optional and also advanced, it seemed more appropriate to add the link toggle in the Advanced panel as well, because it would be used less frequently and also intentionally. Please let me know if this makes any difference.